packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -28,6 +31,7 @@ const SelectedFilters = {
 			return this.$scopedSlots.default({
 				components: this.components,
 				selectedValues: this.selectedValues,
+				componentProps: this.componentProps,
```

Comment:
@mohdashraf010897 There is no need to provide it in default slot.

Reply:
👍